packages/dashboard-message-bus-common/package.json

```diff
@@ -17,7 +17,7 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/trufflesuite/truffle.git",
-    "directory": "packages/dashboard-mesasge-bus"
+    "directory": "packages/dashboard-message-bus"
```

Comment:
Oh hey while you're at it, this should be `dashboard-message-bus-common`, right?

Reply:
You're right, I need 💤